.github/codecov.yml

```diff
@@ -1,7 +1,7 @@
 codecov:
   require_ci_to_pass: false
   notify:
-    wait_for_ci: false
+    wait_for_ci: true
```

Comment:
this is the trick; making codecov wait for all jobs to finish before generating a report.